packages/repository/src/decorators/model.ts

```diff
@@ -15,7 +15,7 @@ export const PROPERTY_KEY = 'loopback:property';
  * @returns {(target:AnyType)}
  */
 export function model(definition?: Object) {
-  return function(target: AnyType) {
+  return function(target: Any) {
```

Comment:
Strangely enough, https://www.typescriptlang.org/docs/handbook/decorators.html shows `target` as `any`. In that case I am proposing to use `any` too, and disable ts-lint warnings.